releasenotes/notes/jruby-detection-ff54fa80eaa0a70e.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Adds JRuby language detection
```

Comment:
May want to expand this description and provide more context for the change. Also can drop the top commented out block.

Reply:
Done